index.md

```diff
@@ -19,6 +20,23 @@ Apply to OceanHackWeek 2025
 ```
 :::
 
+:::{admonition} New in 2025: The OceanHackWeek Fellows Program
+:class: important
```

Comment:
Do we want to use a different admonition level, so it stands out separately?

Various styles: https://pydata-sphinx-theme.readthedocs.io/en/stable/examples/kitchen-sink/admonitions.html

Reply:
I agree with Alex. How about the "admonition" style? Here it is
```suggestion
:class: admonition
```